sections/Stake/ClaimBox/ClaimBox.tsx

```diff
@@ -114,15 +133,28 @@ const ClaimBox: FC<ClaimBoxProps> = ({ stakingContract }) => {
 
```

Comment:
```suggestion
const StyledButton = styled(Button)`
	font-family: ${(props) => props.theme.fonts.ASMRegular};
	font-size: 14px;
`;

```

Reply:
We need to use the Typography component for this if it's necessary because the buttons should be same in the whole site.